fastlane/lib/fastlane/environment_printer.rb

```diff
@@ -293,5 +293,11 @@ def self.copy_to_clipboard(string)
       require 'open3'
       Open3.popen3('pbcopy') { |input, _, _| input << string }
     end
+
+    def self.git_version
+      return `git --version`.strip.split("\n").first
+    rescue
+      return "not found"
```

Comment:
Hmm, maybe we could improve this message a bit:

```
return "git: command not found. You need to make git available on your machine. Please follow the following instructions: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git"
```

WDYT? 🤔

Reply:
I think that'd be more appropriate in the `git_helper` class, so it's visible when using the actions. As this is for debug, and fastlane will run without git for most actions, my worry would be it'd clutter the output.